src/run.py

```diff
@@ -1,58 +1,68 @@
 #add 2 numbers
 def add(x, y):
-    return x + y
+  return x + y
+
 
 #subtract 2 numbers
 def subtract(x, y):
-    return x - y
+  return x - y
+
 
 #multiply 2 numbers
 def multiply(x, y):
-    return x * y
+  return x * y
+
 
 #divide 2 numbers
 def divide(x, y):
-    return x / y
+  return x / y
 
-previousCalc = 'no'
+#variable definitions
+prevCalc = 'no'
+num1 = 0
+num2 = 0
 
+#general start up
 print("Which operation?")
 print("1. Add")
 print("2. Subtract")
 print("3. Multiply")
 print("4. Divide")
```

Comment:
Try not to mix single quotes `'` and double quotes `"` for strings throughout code. I like to stick to single quotes because they look cleaner, and use double quotes for multi-line comments instead.

Reply:
fixed

Reply:
There are still some strings like the `"no"` and `"yes"` that use double quotes in the `prevCalc` comparison